app/views/volunteers/_form.html.slim

```diff
@@ -1,52 +1,54 @@
 = simple_error_notice f
 
 - if policy(Volunteer).supervisor_privileges?
-  .row
-    .col-xs-12
-      = f.input :state, collection: Volunteer.state_collection, default: :registered
-    .col-xs-12
-      = f.input :rejection_type, collection: Volunteer::REJECTIONS, as: :radio_buttons
-    .col-xs-12
-      = f.input :rejection_text
+  fieldset
+    legend= t_attr(:state)
+    .row
+      .col-xs-12
+        = f.input :external, input_html: { data: { state: { remove: ['contacted', 'resigned', 'inactive', 'rejected'], selected: 'accepted' }, hide: ['banc-data', 'checklist'] }, class: 'volunteer-active-checkbox-changes' }
```

Comment:
I think for an external Volunteer only the Personal particulars and adress(mail/phone) is sufficient. Language skills, accompaniments etc is prob. not needed. 
I remember they told us at the meeting that they only would need to have contact info in case to call or mail them.